.vscode/tasks.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Not sure if we want to start adding IDE configuration to our projects. I have got settings for WebStorm and Eclipse for the configurator, but so far I am keeping them in a separate repository and use symlinks to make them accessible inside the project repo.
I think this is the better way of doing it, or else we will start getting a large number of pull requests just to accommodate the various IDEs that are used.

Reply:
I know what you mean, I will remove them and add it to the wiki or something.